modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -147,6 +155,17 @@ repositories {
 	}
 }
 
+if (project.hasProperty("releaseApiJarName")) {
```

Comment:
hey @petershin  this if {} block has to go inside of `repositories` but I think that SF keeps moving it out.